### ISSUE TITLE
acceleration question

### ISSUE THREAD

**Issue by DebugBios** [Ochii/cso2-master-server#90]
The grenade acceleration is now deleted.

Which script should I touch to recover?

**Comment by Ochii** [comment]
Was the grenade acceleration fine in 0.8.2?

**Comment by DebugBios** [comment]
No, the grenade acceleration was deleted with a balance patch before service shutdown. I just want to get it back.

**Comment by dounai2333** [comment]
are you expressing a skill in Zombie Zeta (or TDM Z)? if is, it should be disabled on your community modded client
if you want to recover it, go to "custom\scripts\mod\zombie_zeta\cso2_zs_skills.csv" then find ZZ_SPEED_UP_AFTER_GRENADE, it should has two slashes in start line, delete them to recover
if you don't find it, look again in "Data\cstrike\scripts\mod\zombie_zeta\cso2_zs_skills.ecsv", if you still can't find it, that mean your client hasn't made any changes to this skill, and it's enabled in 24406

if that's not what you want to say, whatever...
